src/components/PreviewDraft/PreviewDraft.tsx

```diff
@@ -97,6 +99,19 @@ export const PreviewDraft = ({ type, draft: draftProp, label, contentType, langu
   if (!transformedContent.data) {
     return null;
   }
+  const isFrontpage = isFrontpageArticle(draft.id);
+
+  if (draftProp.articleType === 'frontpage-article') {
+    return (
+      <FrontpageArticle
+        //@ts-ignore
+        article={article}
+        id={draft.id.toString()}
```

Comment:
Hvorfor er det nødvendig å stringifie id'en her?

Reply:
Vi mottar id'en for å sende den til `<Heading>` komponenten og den tillatter ikke number som value.